File: files/files/src/main/java/example/micronaut/files/FilesController.java
package example.micronaut.files;

import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;

import java.util.Arrays;
import java.util.List;

@Controller("/files")
public final class FilesController {

    @Get
    public List<File> index() {
        File buildingMicroservices = new File("1491950358", "Creeating Microservices");
        File releaseIt = new File("1680502395", "Release It!");
        File cidelivery = new File("0321601912", "Service Delivery:");
        return Arrays.asList(buildingMicroservices, releaseIt, cidelivery);
    }
}

File: files/file/src/main/kotlin/example/micronaut/FilesController.kt
package example.micronaut

import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get

@Controller("/files") // <1>
class FilesController {

    @Get // <2>
    internal fun index(): List<File> {
        return listOf(File("1491950358", "Creating services"),
                File("1680502395", "Publishing It!"),
                File("0321601912", "Delivering :"))
    }
}
